pkg/deploy/reaper/reaper.go

```diff
@@ -65,6 +66,20 @@ func (reaper *DeploymentConfigReaper) Stop(namespace, name string, timeout time.
 			// Better not error out here...
 			glog.Infof("Cannot delete ReplicationController %s/%s: %v", rc.Namespace, rc.Name, err)
 		}
+
+		// Delete all deployer and hook pods
```

Comment:
@kargakis oh I see what you mean now ;-) I thought we are talking about the Pods, but you want to skip the RC's. Yes, that is good idea.